lib/openstax/utilities/access_policy.rb

```diff
@@ -24,7 +24,8 @@ def self.respond_to_missing?(method_name, include_private = false)
       end
 
       def self.require_action_allowed!(action, requestor, resource)
-        raise SecurityTransgression unless action_allowed?(action, requestor, resource)
+        msg = "\"#{requestor.inspect}\" is not allowed to perform \"#{action}\" on \"#{resource.inspect}\""
+        raise SecurityTransgression.new(msg) unless action_allowed?(action, requestor, resource)
```

Comment:
I normally go with the format below -- [the raise call](http://apidock.com/ruby/Kernel/raise) takes a second argument which is the message.

```ruby
raise(SecurityTransgression, msg) unless action_allowed?(action, requestor, resource)
```